src/main/java/org/jabref/logic/preferences/JabRefCliPreferences.java

```diff
@@ -1837,7 +1851,13 @@ public AiPreferences getAiPreferences() {
                 getInt(AI_DOCUMENT_SPLITTER_CHUNK_SIZE),
                 getInt(AI_DOCUMENT_SPLITTER_OVERLAP_SIZE),
                 getInt(AI_RAG_MAX_RESULTS_COUNT),
-                getDouble(AI_RAG_MIN_SCORE));
+                getDouble(AI_RAG_MIN_SCORE),
+                new HashMap<>(Map.of(
```

Comment:
Why?

Reply:
Why wrap a map into another map 😅

Reply:
Ah, you're right. I used `ObservableMap` and `MapProperty` before, and they required `Map` to be modifiable.

But I couldn't configure listeners properly. Then I just used `Map<.., Property<...>>`...